.github/workflows/build.yml

```diff
@@ -107,21 +103,26 @@ jobs:
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
   test:
-    runs-on: ubuntu-latest
+    name: test source - py${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os:
+          - ubuntu-latest
         python-version:
-          - "2.7"
-          - "3.6"
           - "3.7"
           - "3.8"
           - "3.9"
           - "3.10"
+          - "3.11"
+        include:
+          - os: ubuntu-20.04
```

Comment:
Why is this necessary?

Reply:
https://github.com/cisagov/skeleton-python-library/pull/114

Reply:
Ah, right.